hawkbit-repository/hawkbit-repository-jpa/src/main/java/org/eclipse/hawkbit/repository/jpa/model/JpaTargetFilterQuery.java

```diff
@@ -77,6 +77,9 @@
     @Column(name = "auto_assign_weight", nullable = true)
     private Integer autoAssignWeight;
 
+    @Column(name = "auto_assign_triggered_by", nullable = true, length = 64)
+    private String autoAssignTriggeredBy;
```

Comment:
Not sure if this is a good wording here. The trigger is the scheduler/system. Isn't more about who configured the auto assignment?

Reply:
@bogdan-bondar do you have a better suggestion?

Reply:
would `auto_assign_enabled_by` fit more here?

Reply:
`autoAssignInitiatedBy`? Same can be used for the action

Reply:
sounds good for me, do you agree @schabdo ?

Reply:
Yes, thanks @bogdan-bondar